TROUBLESHOOTING.md

```diff
@@ -0,0 +1,23 @@
+# Troubleshooting
```

Comment:
Let's just put this into `ci-operator/jobs` and make it specific, like `MERGE_BLOCKING_STATUS_MIGRATION.md`